client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -315,6 +315,10 @@ export default React.createClass( {
 	},
 
 	newDomains() {
+		if ( get( this.props, 'selectedSite.options.is_domain_only' ) ) {
```

Comment:
What about `connect` and using proper selectors so we don't have to refactor later? :)